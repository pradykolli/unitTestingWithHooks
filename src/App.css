.App {
  text-align: center;
}

.App-logo {
  height: 40vmin;
  pointer-events: none;
}

@media (prefers-reduced-motion: no-preference) {
  .App-logo {
    animation: App-logo-spin infinite 20s linear;
  }
}

.App-header {
  background-color: #282c34;
  min-height: 100vh;
  font-size: calc(10px + 2vmin);
  color: white;
}

.App-link {
  color: #61dafb;
}
.wrapper{
  width:100%;
}
.division{
  width:31%;
  margin-left: 1%;
  margin-right: 1%;
  box-shadow: 4px 4px 8px 0 #999,
              -4px -4px 8px 0 #666;
  border-radius: 5px;
  float: left;
}
.buyButton{
  width: 100%;
  height: 30px;
  border-radius: 25px;
  color: #fff;
  background: red;
  margin-bottom: 5px;
  border: none;
}
.addButton{
  width: 100%;
  height: 30px;
  border-radius: 25px;
  color: #fff;
  background: limegreen;
  margin-bottom: 5px;
  border: none;

}
@keyframes App-logo-spin {
  from {
    transform: rotate(0deg);
  }
  to {
    transform: rotate(360deg);
  }
}
