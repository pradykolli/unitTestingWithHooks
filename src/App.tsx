import React from 'react';
import './App.css';
import Cafe from './Components/Cafe/Cafe';

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <Cafe />
      </header>
    </div>
  );
}

export default App;
