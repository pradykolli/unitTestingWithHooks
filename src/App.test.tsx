import React from 'react';

import Enzyme,{shallow, mount} from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import App from './App';
Enzyme.configure({ adapter: new Adapter() })
describe("App component renders", () => {
  it('app component Matches SnapShot', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});

