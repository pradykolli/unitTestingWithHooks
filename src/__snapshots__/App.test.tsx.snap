// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App component renders app component Matches SnapShot 1`] = `ShallowWrapper {}`;
